lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
```

Comment:
`now` is already calculated and `(now - that.$started) / 1000` is used a few lines above, makes sense to save the value there and reuse it here